luxtronik/datatypes.py

```diff
@@ -378,6 +378,20 @@ def from_heatpump(cls, value):
         return "".join([chr(c) for c in value]).strip("\x00")
 
 
+class MajorMinorVersion(Base):
+    """MajorMinorVersion datatype, converts from and to a RBEVersion"""
+
+    datatype_class = "version"
+
+    @classmethod
+    def from_heatpump(cls, value):
+        if value > 0:
+            major = value // 100
+            minor = value % 100
+            return ".".join(str(x) for x in [major, minor])
```

Comment:
Is there any advantage over this vs using an f-string?

I find f-string much more readable, something like `f"{major}.{minor}"`

Reply:
Nah, you're right. There is absolutely no advantage in my more complicated approach, somehow just didn't have f strings at mind. Was too much inspired by the code from `Version` and haven't given it another thought. Addressed with 95df02d.